spec/less-spec.coffee

```diff
@@ -350,22 +350,69 @@ describe "Less grammar", ->
+      expect(tokens[20]).toEqual value: 'vh', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'meta.function.calc.css', 'constant.numeric.css', 'keyword.other.unit.vh.css']
+      expect(tokens[22]).toEqual value: '-', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'meta.function.calc.css', 'keyword.operator.arithmetic.css']
+      expect(tokens[24]).toEqual value: '40', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'meta.function.calc.css', 'constant.numeric.css']
+      expect(tokens[25]).toEqual value: 'px', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'meta.function.calc.css', 'constant.numeric.css', 'keyword.other.unit.px.css']
+      expect(tokens[27]).toEqual value: '+', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'meta.function.calc.css', 'keyword.operator.arithmetic.css']
+      expect(tokens[29]).toEqual value: '5', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'meta.function.calc.css', 'constant.numeric.css']
+      expect(tokens[30]).toEqual value: '%', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'meta.function.calc.css', 'constant.numeric.css', 'keyword.other.unit.percentage.css']
+      expect(tokens[32]).toEqual value: ',', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'punctuation.separator.list.comma.css']
+      expect(tokens[34]).toEqual value: 'red', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'support.constant.color.w3c-standard-color-name.css']
+      expect(tokens[35]).toEqual value: ';', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'punctuation.separator.parameter.less']
+      expect(tokens[37]).toEqual value: '@', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'variable.parameter.less', 'punctuation.definition.variable.less']
+      expect(tokens[38]).toEqual value: 'b', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'variable.parameter.less']
+      expect(tokens[39]).toEqual value: ';', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'punctuation.separator.parameter.less']
+      expect(tokens[41]).toEqual value: '@', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'variable.parameter.less', 'punctuation.definition.variable.less']
+      expect(tokens[42]).toEqual value: 'c', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'variable.parameter.less']
+      expect(tokens[43]).toEqual value: ')', scopes: ['source.css.less', 'meta.mixin.less', 'meta.parameters.less', 'punctuation.definition.parameters.end.bracket.round.less']
+      expect(tokens[44].scopes).not.toContain 'meta.parameters.less'
```

Comment:
Try to avoid the `.not` qualifier when writing language tests, because scopes can and do change often.  Then if the _new_ `meta.parameters.less` is added back this test will still pass.  While more verbose, it's more useful to write out the exact scope list (i.e. just like tokens[43]).

Reply:
Got it. Fixed.